cfn-templates/cid-cfn.yml

```diff
@@ -4,67 +4,94 @@ Metadata:
   AWS::CloudFormation::Interface:
     ParameterGroups:
       - Label:
-          default: 'Prerequisites'
+          default: 'Common Parameters'
         Parameters:
           - PrerequisitesQuickSight
-          - PrerequisitesCUR
           - PrerequisitesQuickSightPermissions
-      - Label:
-          default: 'Mandatory parameters'
-        Parameters:
           - QuickSightUser
-          - CURBucketPath
-          - CURIsManagedByCloudFormation
       - Label:
-          default: Available dashboards. Keep all 'no' for the first deployment. Go to QuickSight, change Permission and after that update the stack putting to 'yes' 3 first dashboards.
+          default: CUDOS Cost-Intelligence-Dashboard and KPI-Dashboard. Require deployment of CUR via CloudFormation (cur-aggregation.yaml) or manually 24h before.
         Parameters:
+          - CURBucketPath
           - DeployCUDOSDashboard
           - DeployCostIntelligenceDashboard
           - DeployKPIDashboard
+      - Label:
+          default: Trusted Advisor and Compute Optimizer Dashboards. To deploy theese two dashboard, you must first deploy the Optimization Data Collection Lab (https://wellarchitectedlabs.com/cost/300_labs/300_optimization_data_collection/)
+        Parameters:
+          - OptimizationDataCollectionBucketPath
           - DeployTAODashboard
           - DeployComputeOptimizerDashboard
-          - CostOptimizationDataCollectionBucketPath
       - Label:
-          default: 'Optional Parameters (do not change unless you know why)'
+          default: 'Technical Parameters. Please do not change.'
         Parameters:
           - AthenaWorkgroup
           - AthenaQueryResultsBucket
-          - CURDatabaseName
+          - DatabaseName
           - CURTableName
           - CidVersion
           - Suffix
           - QuickSightDataSetRefreshSchedule
           - LambdaLayerBucketPrefix
+    ParameterLabels: 
+      PrerequisitesQuickSight:
+        default: "I have enabled QuickSight Enterprise Edition AND I have a SPICE capacity in the current region."
+      PrerequisitesQuickSightPermissions:
+        default: "I understand that I need to manually give Permission to QuickSight to access CUR bucket and Query results bucket. Then manually refresh datasets after deploying this CFN."
```

Comment:
Customers should do that before deployment of this CFN, hence I suggest to re-phrase to:
I have provided permissions to QuickSight to access CUR bucket and Athena query results bucket.

Reply:
Providing access to suery bucket is not possible as it is not created before